### DISCUSSION TITLE
using honeypot for spam protection

### DISCUSSION THREAD

**Discussion by pcfjojo** [livewire/livewire#2039] [1. Help]
What is the better way to use this package 
[https://github.com/spatie/laravel-honeypot](url) for spam control?

**Comment by Jul000** [comment]
I m searching a solution too. It would be great if we could use honeypot with livewire... Thanks

**Reply by joshhanley** [reply]
@Julien0510 the package would have to be updated to support Livewire. I notice they have both Laravel and Inertia support, so it might be worth raising an issue on that package 🙂
